pkg/crds/dashboard/crds.go

```diff
@@ -46,6 +46,13 @@ func List(cfg *rest.Config) (_ []crd.CRD, err error) {
 			c.GVK.Version = "v1"
 			return c
 		}),
+		newCRD(&v3.PodSecurityAdmissionConfigurationTemplate{}, func(c crd.CRD) crd.CRD {
+			c.Status = true
```

Comment:
I don't believe we need Status subresource for this, so it should probably be `false`. Did it require `true` in your testing for some reason?

Reply:
No, I've removed it.